setup.cfg

```diff
@@ -52,8 +52,10 @@ dev =
     kivy_deps.glew_dev~=0.3.1; sys_platform == "win32"
     flake8
     pre-commit
+    responses
 base =
     pillow
+    requests
     docutils
```

Comment:
I think it's ok. requests is now a hard dependency because you didn't make the import conditional, it could be done, but at this point most users will have it directly or through dependencies anyway.